test/system/admin/feature_flags_management_test.rb

```diff
@@ -0,0 +1,19 @@
+require "application_system_test_case"
+
+class FeatureFlagsManagementTest < ApplicationSystemTestCase
+  setup do
+    sign_in_as(:admin)
+    visit root_path
+    click_link "Nastavenia"
+    click_link "Feature flags"
```

Comment:
Toto treba nejako prelozit do SK. -

 Tiez rozmyslam ci to niekde viac neschovat.